fission/src/mirabuf/MirabufLoader.ts

```diff
@@ -25,22 +31,40 @@ export interface MirabufRemoteInfo {
 
 type MapCache = { [id: MirabufCacheID]: MirabufCacheInfo }
 
-const robotsDirName = "Robots"
-const fieldsDirName = "Fields"
+const robotsDirName = "robot"
+const fieldsDirName = "field"
+const piecesDirName = "piece"
 const root = await navigator.storage.getDirectory()
-const robotFolderHandle = await root.getDirectoryHandle(robotsDirName, { create: true })
-const fieldFolderHandle = await root.getDirectoryHandle(fieldsDirName, { create: true })
+const getDirectoryHandle = (dirName: string) => root.getDirectoryHandle(dirName, { create: true })
+
+const dirNameMap: Record<MiraType, string> = {
+    [MiraType.ROBOT]: robotsDirName,
+    [MiraType.FIELD]: fieldsDirName,
+    [MiraType.PIECE]: piecesDirName,
+}
+
+const dirHandleMap: Record<MiraType, FileSystemDirectoryHandle> = {
+    [MiraType.ROBOT]: await getDirectoryHandle(robotsDirName),
+    [MiraType.FIELD]: await getDirectoryHandle(fieldsDirName),
+    [MiraType.PIECE]: await getDirectoryHandle(piecesDirName),
+}
```

Comment:
Using records instead of ternaries was a refactor caused by this PR, since double ternaries are ugly.